crates/database/src/order_execution.rs

```diff
@@ -2,53 +2,32 @@ use {
     crate::{auction::AuctionId, OrderUid},
     bigdecimal::BigDecimal,
     sqlx::PgConnection,
-    std::ops::DerefMut,
 };
 
 pub async fn save(
     ex: &mut PgConnection,
     order: &OrderUid,
     auction: AuctionId,
-    surplus_fee: Option<&BigDecimal>,
-    scoring_fee: Option<&BigDecimal>,
+    executed_fee: &BigDecimal,
 ) -> Result<(), sqlx::Error> {
+    let scoring_fee: Option<&BigDecimal> = None; // we don't need a scoring fee anymore
```

Comment:
Can we just omit this column when populating the query?

Reply:
Sure. Cleaned up.